apps/condo/domains/user/schema/AuthenticateOrRegisterUserWithTokenService.js

```
@@ -395,10 +388,12 @@ const AuthenticateOrRegisterUserWithTokenService = new GQLCustomSchema('Authenti
                     actualUser = existingUser
 
                     if (Object.keys(userPayload).some(key => key !== 'dv' && key !== 'sender')) {
-                        actualUser = await User.update(context, existingUser.id, userPayload, 'id', {
+                        const updateUserData = { ...userData }
+                        delete updateUserData['country']
+                        actualUser = await User.update(context, existingUser.id, updateUserData, 'id', {
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)